<Type Name="ServerBusyException" FullName="Microsoft.Azure.EventHubs.ServerBusyException">
  <TypeSignature Language="C#" Value="public sealed class ServerBusyException : Microsoft.Azure.EventHubs.EventHubsException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed beforefieldinit ServerBusyException extends Microsoft.Azure.EventHubs.EventHubsException" />
  <TypeSignature Language="DocId" Value="T:Microsoft.Azure.EventHubs.ServerBusyException" />
  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class ServerBusyException&#xA;Inherits EventHubsException" />
  <TypeSignature Language="F#" Value="type ServerBusyException = class&#xA;    inherit EventHubsException" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.Azure.EventHubs</AssemblyName>
    <AssemblyVersion>1.0.3.0</AssemblyVersion>
    <AssemblyVersion>2.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>Microsoft.Azure.EventHubs.EventHubsException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
            <span data-ttu-id="cb6b0-101">サーバーがビジー状態である場合にスローされる例外。</span><span class="sxs-lookup"><span data-stu-id="cb6b0-101">The exception that is thrown when a server is busy.</span></span>  <span data-ttu-id="cb6b0-102">呼び出し元は、しばらく待つし、操作をやり直してください必要があります。</span><span class="sxs-lookup"><span data-stu-id="cb6b0-102">Callers should wait a while and retry the operation.</span></span>
            </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members />
</Type>